tests/puppeteer/pages/BO/international/taxes/taxRules/add.js

```diff
@@ -0,0 +1,70 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddTaxRules extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Tax Rules > Add new • prestashop';
```

Comment:
```suggestion
    this.pageTitleCreate = 'Tax Rules > Add new •';
```
depends of installation values